### ISSUE TITLE
How to include space in the path to dictionary?

### ISSUE THREAD

**Issue by yuyuxing-wu** [SamuraiT/mecab-python3#43]
I want to include space in path and have tried like below.

````
import MeCab
parser=MeCab.Tagger('-d /opt/test1 test2/dictionary')
parser=MeCab.Tagger('-d /opt/test1\ test2/dictionary')
parser=MeCab.Tagger('-d "/opt/test1 test2/dictionary"')
````

All three of them failed as RuntimeError.

Is any way to include space in path?

**Comment by polm** [comment]
This is not possible right now because the argument handling doesn't do full shell quoting; see #25 for details. 

I haven't thought about this problem in a long time, but I'll take another look at whether there's a good way to handle it. It may be possible to just preparse argument with Python's `shlex`.

**Comment by polm** [comment]
So I took a look at it and this issue should be fixed in master. Unfortunately that broke Node-based parsing for some reason. Should be fixable at least, though can't say how long it'll take.

**Comment by polm** [comment]
OK, fixed the issues, so this should be resolved in the latest release candidate. 

Could you please install the latest version like below and tell me if it fixes your issue?

    pip install mecab-python3=0.996.6rc2

**Comment by yuyuxing-wu** [comment]
Thanks!

I have tried two patterns like below and both of them works!

```
import MeCab
parser=MeCab.Tagger('-d /opt/test1\ test2 -u /opt/test1\ test2/user.dic')
parser=MeCab.Tagger('-d "/opt/test1 test2" -u "/opt/test1 test2/user.dic"')
```

**Comment by polm** [comment]
OK, thanks for the confirmation. I'll release a new update with this change soon.

**Comment by yuyuxing-wu** [comment]
FYI:
The escaped character also need to be changed
from 
'-F%M\\t%H\\n'
to 
'-F%M\t%H\n'

**Comment by polm** [comment]
Fix for this is included in the 1.0 release today.
